PlayCover/Views/MainView.swift

```diff
@@ -23,11 +23,15 @@ struct MainView: View {
     @State private var viewWidth: CGFloat = 0
     @State private var collapsed: Bool = false
     @State private var showSourceFolders = true
+    @State private var showAppFolders = true
     @State private var selectedBackgroundColor: Color = Color.accentColor
     @State private var selectedTextColor: Color = Color.black
+    @State private var addFolderPresented = false
+    @State var newFolder = ""
+    @StateObject var foldersObject = AppFolder()
```

Comment:
After `AppFolder` is converted to `ObservableObject`, this should be changed to `ObservedObject`.